server/plugin/api.go

```diff
@@ -85,13 +85,14 @@ func (p *MatterpollPlugin) InitAPI() *mux.Router {
 	r.HandleFunc("/"+iconFilename, p.handleLogo).Methods("GET")
```

Comment:
Would you change `"GET"` to `http.MethodGet` for consistency?

Reply:
Good catch :+1: